### ISSUE TITLE
You can edit other's translations

### ISSUE THREAD

**Issue by Steffan153** [Codewars/codewars.com#1938]
On anyone's translation, if you remove the `?sel` part from the URL and add `/edit` you can edit it. For example, on [this](https://www.codewars.com/kumite/5bd22fc87e21b82e9f0001cf?sel=5bd22fc87e21b82e9f0001cf) translation, I can edit it [here](https://www.codewars.com/kumite/5bd22fc87e21b82e9f0001cf/edit).

**Comment by error256** [comment]
Isn't there an error when you try to publish it? That's how it was.

**Comment by Steffan153** [comment]
Oh, I didn't try that. :thinking: 
No error, though, it looks like this (it does nothing tho) :confused::
![DeepinScreenshot_select-area_20190910164242](https://user-images.githubusercontent.com/40404519/64656463-c0173600-d3ec-11e9-9772-a1518b7fca7d.png)

**Comment by Voileexperiments** [comment]
Yes, you can still access the endpoint but the data failed to load.

You used to able to open new translation page without the right either because the endpoint is not protected, not sure if it has been fixed.

**Comment by Steffan153** [comment]
ok... I'll reopen this then.
